SportHub/src/modules/SearchResultsList/components/SearchResultsList.js

```diff
@@ -20,6 +21,7 @@ const SearchResultsList = ({ contentSearchValue }) => {
   const [refreshSearch, setRefreshSearch] = useState(false)
 
   const [articles, setArticles] = useState([])
+  const [numberOfArticles, setNumberOfArticles] = useState()
```

Comment:
[NEED TO BE CONSIDERED] I would suggest set default to 0 in case if backend wouldn't respond